Classes/Emogrifier.php

```diff
@@ -911,13 +911,14 @@ private function attributeValueIsImportant($attributeValue)
      *
      * @return void
      */
-    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css)
+    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css, $imports = '')
```

Comment:
This pull request does not change the calls of this method, i.e, $imports will always be an empty string. So there will be no change in behavior. Or am I missing something?